docusaurus/docs/how-to-guides/data-source-plugins/convert-a-frontend-datasource-to-backend.md

```diff
@@ -17,7 +17,7 @@ To convert the frontend data source, we recommend scaffolding a new backend data
 
 ## Why
 
-There are multiple features available only in backend plugins, such as Grafana Alerting or Recorded queries. See the use cases for implementing a backend plugin in the [backend plugins introduction](../../key-concepts/backend-plugins/#use-cases-for-implementing-a-backend-plugin).
+There are multiple features available only in backend plugins, such as Grafana Alerting, Recorded queries or externally shared dashboards (previously called Public dashboards). See the use cases for implementing a backend plugin in the [backend plugins introduction](../../key-concepts/backend-plugins/#use-cases-for-implementing-a-backend-plugin).
```

Comment:
I would just leave "externally shared dashboards" and drop the "previously called Public dashboards)" - no need to list out all the previous names from my pov.

btw. we probably should rename it here too then: https://grafana.com/grafana/plugins/all-plugins/?features=public_dashboards

Reply:
Yes, we should. I'm curious about how that list is filtered 🧐
There are many plugins in that section with public_dashboards filtered that are not part of confirmed data sources here https://grafana.com/docs/grafana-cloud/visualizations/dashboards/share-dashboards-panels/shared-dashboards/#supported-data-sources